Rakefile

```diff
@@ -63,6 +65,15 @@ def replace_header(head, header_name)
   head.sub!(/(\.#{header_name}\s*= ').*'/) { "#{$1}#{send(header_name)}'"}
 end
 
+def bundle_katex_fonts(assets_path)
```

Comment:
I wasn't able to succesfully the fonts using sprockets: adding `*.woff2` to the sprockets manifest would result in the fonts being saved to `lib/gollum/public/assets/katex/dist/fonts` as expected, but since they would be managed the would have filesnames of the form: `KaTeX_Fontname-SPROCKETSHASH.woff2`. However, the KaTeX CSS is unware of our sprockets setup and would still look for the 'normal' filename `KaTeX_Fontname.woff2`, which did not exist.

The solution here is to just copy the fonts manually at the end of the precompilation task, thus preserving the original filenames.

An alternative could be to override the font definitions in our own style files, but that would also be a bit of a hack, and possible a more difficult one to maintain.

Reply:
Also note that I'm only copying the `.woff2` font files to save space. Given [woff2 support](https://caniuse.com/woff2) these days I think this should be ok?

Reply:
> Also note that I'm only copying the `.woff2` font files to save space. Given [woff2 support](https://caniuse.com/woff2) these days I think this should be ok?

Yes, at work I only need to support browsers that support `woff2` these days. If the corporate overlords are okay with that, maybe we can be too!

Reply:
Regarding fingerprinted/digested assets: that is too bad about the fonts. But fonts don't change that often, so maybe your solution is okay long term.

If we run into this a lot, we may be able to find or create a Sprockets extension that gives us the ability to un-fingerprint certain types of assets—like fonts.

Reply:
> The solution here is to just copy the fonts manually at the end of the precompilation task, thus preserving the original filenames.

Could we rename them instead of copying over them? Seems computationally cheaper, though it may not matter that much. Just a thought.

Reply:
> If we run into this a lot, we may be able to find or create a Sprockets extension that gives us the ability to un-fingerprint certain types of assets—like fonts.

Good idea to keep in the back of our minds if we're not satisfied with this in the long run!

> Could we rename them instead of copying over them?

We could but then we'd be breaking the source installation of the library inside `node_modules`. If you then run gollum with `--development-assets` (which serves the assets from `node_modules`) it would break. :)